doc/source/appendix/json_intro.rst

```diff
@@ -61,114 +61,10 @@ Here is a serialized variant in JSON. It's a bit of an edge case in some respect
 
 Things to notice:
  * A serialized record contains no explicit information about its type.
- * Arrays are serialized as JSON arrays.
+ * "repeated" types are serialized as JSON arrays.
  * Maps are serialized as JSON objects.
  * Records are also serialized as JSON objects.
  * Enums (not shown here) are serialized as JSON strings.
  * Nulls are serialized as JSON nulls.
```

Comment:
I would rm. This bullet is misleading. This would only apply to message types, and the the spec allows this, but it's not required. I think by default this will not be the case, i.e. the field will just be omitted entirely rather than included as a JSON null.